src/components/calendars/CalendarView.js

```diff
@@ -0,0 +1,81 @@
+import AddIcon from "@mui/icons-material/Add";
+import {
+  Button,
+  Container,
+  Dialog,
+  DialogContent,
+  Grid,
+  Skeleton,
+  Stack,
+  Typography,
+} from "@mui/material";
+import React, { useState } from "react";
+
+import AddCalendar from "./AddCalendar";
+import CalendarCard from "./CalendarCard";
+import useCalendars from "./useCalendars";
+
+const CalendarsView = () => {
+    const { calendars, loading, refreshCalendars } = useCalendars();
+    const [isAddCalendarFormOpen, setIsAddCalendarFormOpen] = useState(false);
+
+    const renderCalendarSection = (calendars) => {
+        return (
+            <Grid container spacing={2} sx={{ mt: 2 }}>
+                {calendars.map((calendar) => (
+                    <CalendarCard key={calendar.key} calendar={calendar} />
+                ))}
+            </Grid>
+        );
+    }
+
+    const handleAddCalendarClick = () => {
+        setIsAddCalendarFormOpen(true);
+    };
+
+    const handleAddCalendarFormClose = () => {
+        setIsAddCalendarFormOpen(false);
+        refreshCalendars();
```

Comment:
maybe only refresh calendar after user actually add something that worth refreshing